src/server/authserver/Main.cpp

```diff
@@ -113,7 +113,25 @@ extern int main(int argc, char** argv)
 
     sLog->outString("%s (authserver)", _FULLVERSION);
     sLog->outString("<Ctrl-C> to stop.\n");
-    sLog->outString("Using configuration file %s.", configFile);
```

Comment:
@ShinDarth Fix what? What mistakes this time? The code was infuncional? You have errors in time to make the compilation? What is the problem this time in the code?